src/tactic/rewrite_search/discovery.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2020 Kevin Lacker, Keeley Hoek, Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Lacker, Keeley Hoek, Scott Morrison
+-/
+import tactic.nth_rewrite
+import tactic.rewrite_search.types
+
+/-!
+# Generating a list of rewrites to use as steps in rewrite search.
+-/
+
+namespace tactic.rewrite_search
+
+open tactic tactic.interactive tactic.rewrite_search
+
+/-
```

Comment:
Here and below, doc string comments should start with `/--` in order to show up when you hover over them and for them to show up on the website.
```suggestion
/--
```

Reply:
The private methods should not be used by anyone outside this file (well, if I understand Lean method visibility correctly they *cannot* be used by anyone outside this file), so I think they should not have docstrings that should show up on the website. This is the rule that the linter enforces, too, that private methods don't require docstrings, & this does pass the lint checks. I added regular comments so that there is still an explanation of what they do, for users who are in the same file they are defined.

Reply:
Turning the comments into docstrings will make things more convenient for people working in that file by letting them see the comments simply by hovering over the declaration names, rather than scrolling up. Also, the declarations of the private functions are still going to show up in the online docs, and I see no reason why the comments about them shouldn't appear with them as well.

Reply:
All right, I changed them to docstrings.